.github/workflows/_measure-gas-and-contract-size.yml

```diff
@@ -39,14 +42,29 @@ jobs:
       - name: Generate all contracts
         run: make generate-contracts
 
+      - name: Compile all contracts
+        run: make compile-contracts
+
       - name: Run gas-consumption binary on the current branch
         run: |
           export CONTRACTS_DIR="contracts"
           export CARGO_MANIFEST_DIR=./Cargo.toml
-          ./target/release/gas-consumption current-report.txt
+          ./target/release/gas-consumption current-gas.txt
 
       - name: Print gas measured on the current branch
-        run: cat current-report.txt
+        run: cat current-gas.txt
+
+      - name: Measure verifiers code size
+        run: |
+          echo -n "Deposit: " > current-size.txt
+          cat ./artifacts/DepositVerifier.sol/Halo2Verifier.json \
+            | jq -r '.bytecode.object' | wc -c >> current-size.txt
```

Comment:
Maybe divide by 2 here to get size in bytes, because it's currently it's len of hex string.

Reply:
done